config/env_sample.json

```diff
@@ -206,6 +206,25 @@
     "/*  DATA_WAREHOUSE_PORT=5432": "*/",
     "/*  Enable/Disable Unizin Date Warehouse specific features/data": "*/",
     "DATA_WAREHOUSE_IS_UNIZIN": true,
+    "/*  Learning Record Store configuration": "*/",
+    "/*  Uncomment these variables and fill them in if you're using a ": "*/",
+    "/*  non-bigquery LRS source. These are optional": "*/",
+    "/*  LRS database engine driver": "*/",
+    "LRS_ENGINE": "django.db.backends.postgresql",
```

Comment:
We could **probably** just make this variable `google.cloud.bigquery` rather than have the boolean "RESOURCE_USE_LRS"

Reply:
Fixed